Dockerfile

```diff
@@ -14,13 +13,24 @@ RUN apt-get update && apt-get install ca-certificates && \
     apt-get update && \
     apt-get install -y yarn
 
-WORKDIR /data/
+RUN echo ${VERSION} > /version.txt
+
+WORKDIR /src/
+#get dependencies
+COPY package.json .
+COPY yarn.lock .
+COPY static .
+RUN yarn install
 
-# Build for static resources
-RUN git clone https://github.com/logseq/logseq.git &&  cd /data/logseq && yarn && yarn release && mv ./static ./public && rm -r ./public/workspaces
+# copy everything else & run release
+COPY . .
```

Comment:
Don't quite understand the purpose here

Reply:
me neither, didn't make sense to build a random commit (the one at time when the docker image is built) this just copies everything in the repository (excluding the ignored files)